src/styles/_stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    margin-right: 4px;
+
+    &:nth-child(-n + 4) {
+      background-image: url('../images/star-active.svg');
+    }
```

Comment:
The nth-child selector here will always select the first four child elements, regardless of the parent's class. This might not be the desired behavior if you want to create a rating system where the number of active stars can change. Consider moving this logic to the parent class, and creating separate classes for different numbers of active stars.